.github/workflows/ci.yaml

```diff
@@ -17,6 +17,7 @@ env:
   PIP_CACHE: /tmp/pip-cache
   SQLALCHEMY_WARN_20: 1
   PYTHONASYNCIODEBUG: 1
+  DEPENDENCY_CONFLICTS: 14  # Update this one if a PR resolves a dependency conflict
```

Comment:
I don't think we should require people to change workflows what fixing things...

Reply:
I don't like that part much either. What would you recommend?
* Don't require people to update it. That would leave the door open for regressions.
* Define it as repository secret.
* Define it in `script/pip_check`. IMO the next best option.
* Something else?

In the end, this is just for the last `14` conflicts. So nothing that happens regularly.

Reply:
Yeah, the pip check script would be the next best place I guess.

Reply:
Done in https://github.com/home-assistant/core/pull/64058/commits/e852c11c9d98b0d833bec723fefcfe6f649e9798